CMakeLists.txt

```diff
@@ -169,6 +120,11 @@ if(BUILD_TESTING)
   enable_testing()
 endif()
 
+# plugins
+option(F3D_PLUGINS_STATIC_BUILD "Make all plugins static (embedded into libf3d)" OFF)
```

Comment:
i think this should be on by default

Reply:
looping back on this, I'm trying to think how will need this on or off.

- Building for a redistributable (like our CI/superbuild) : ON
- Building for a package manager: OFF
- Building for local usage: ON

I think a default value of ON makes sense.

Reply:
Ok, I wasn't sure, but if you prefer enabled by default I'm fine.